docs/reference/api/platform/Callback.md

```diff
@@ -27,3 +27,156 @@ Here is an example that uses everything discussed in the [introduction to callba
 
 - [Platform overview](/docs/development/reference/platform.html).
```

Comment:
This and Intro + related content should now be deleted

Reply:
Thanks! I'll do that.